sources/whippet/Cargo.toml

```
@@ -0,0 +1,25 @@
+[package]
+name = "whippet"
+version = "0.1.0"
```

You can give yourself some credit here 😄 

```suggestion
version = "0.1.0"
authors = [ "..." ]
```